operator/pkg/client/client_factory.go

```diff
@@ -103,11 +104,24 @@ func NewSingletonClients(config *rest.Config, logger logr.Logger) (*SingletonCli
 	discoveryRESTMapper := restmapper.NewDeferredDiscoveryRESTMapper(cachedDiscoveryClient)
 	discoveryShortcutExpander := restmapper.NewShortcutExpander(discoveryRESTMapper, cachedDiscoveryClient)
```

Comment:
Didnt you mention we dont need discoveryRESTMapper anymore for the expander?

Reply:
We do but it is not required to be exposed. Expander hold the mapper instance in the background.